src/core/Akka.Streams/Dsl/Source.cs

```diff
@@ -134,6 +134,14 @@ public IRunnableGraph<TMat> To<TMat2>(IGraph<SinkShape<TOut>, TMat2> sink)
             return new Source<TOut, TMat2>(Module.TransformMaterializedValue(mapFunc));
         }
 
+        internal Source<TOut2, TMat> DeprecatedAndThen<TOut2>(StageModule op)
```

Comment:
It's still there in the latest version, and I'm not sure which direction they're taking.